JavaScript/Github_unfollowers/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Please remove the `.gitignore`. We have a global `.gitignore` to cover everything up :)